tools/app-provisioning-tool/vs2019-16.9-how-to-use.md

```diff
@@ -0,0 +1,90 @@
+# Creating ASP.NET Core projects with Microsoft identity platform(Azure AD/B2C).
+
+In the 16.9 release of [Visual Studio](https://visualstudio.microsoft.com/downloads/), we updated the ASP.NET Core templates to allow you to use the Microsoft identity platform for authentication. We are still working to provide an experience where you can specify the auth options in Visual Studio 2019 and have it registered with the Microsoft identity platform and configure the generated project. With the new auth option, Microsoft identity platform, in the 16.9 release. You can create projects that are setup to for authentication, but there will be some additional work that you'll need to perform to complete the process. This document walks you through the available options . In a later update to Visual Studio we will streamline this experience so that all steps can be performed in Visual Studio itself.
```

Comment:
```suggestion
In the 16.9 release of [Visual Studio](https://visualstudio.microsoft.com/downloads/), we updated the ASP.NET Core templates to allow you to use the Microsoft identity platform for authentication. We are still working to provide an experience where you can specify the auth options in Visual Studio 2019 and have it registered with the Microsoft identity platform and configure the generated project. With the new auth option, Microsoft identity platform, in the 16.9 release, you can create projects that are setup for authentication, but there will be some additional work that you'll need to perform to complete the process. This document walks you through the available options . In a later update to Visual Studio we will streamline this experience so that all steps can be performed in Visual Studio itself.
```